flatdata-generator/tests/generators/cpp_expectations/enums/implicit_values.h

```diff
@@ -9,5 +9,8 @@ enum class Enum1 : uint16_t
     // = 1
     VALUE_4 = 1,
     // = 2
-    VALUE_5 = 2
-};
\ No newline at end of file
+    VALUE_5 = 2,
+    UNKNOWN_VALUE_5 = 5,
```

Comment:
One think which I still don't quite get. Wouldn't it be better just to generate a single UNKNOWN variant and map all unknown value to it?

Reply:
This is done for performance reasons: This way reading from the field is for-free: Just a transmute without any branches

Reply:
Thanks for explaining. Now, I see it better. Following the 0-overhead rule we have to do this.